src/main/java/com/google/devtools/build/lib/bazel/rules/android/android_sdk_repository_template.txt

```diff
@@ -80,4 +80,7 @@ create_system_images_filegroups(
 )
 
 exports_files([
+    'tools/bin/apkanalyzer',
+    'tools/bin/avdmanager',
```

Comment:
apkanalyzer and avdmanager are shell scripts that invoke their corresponding java application. In particular, they set up the classpath for particular jars in the tools/lib directory. So I'm not sure it would be enough to export just the shell scripts. Have you tried using these exported scripts as an action or in a genrule?

What you may need to do is write java_import rules for the jars that are set in the classpath in the scripts, and write a java_binary rule that specifies the main class and depends on the java_imports

Reply:
i tired and i will need to add for apk analyzer I will need to pass in JAVA_HOME env variable in bazel command to make it work